xbmc/settings/lib/SettingsManager.h

```diff
@@ -92,7 +90,7 @@ class CSettingsManager : public ISettingCreator, public ISettingControlCreator,
    \param root XML node
    \return True if the setting values were successfully saved, false otherwise
    */
-  bool Save(TiXmlNode *root) const override;
+  bool Save(TiXmlNode *root) const;
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

Reply:
See #17062.